__tests__/context.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { jest, test, expect } from "@jest/globals";
+
+import { ConsoleContext } from "../src/context";
+
+(global as any).console = {log: jest.fn()};
+
+test("ConsoleContext", () => {
+	const proxy = new ConsoleContext(6, "=>");
+
+	proxy.warn("test");
+	expect(console.log).toHaveBeenLastCalledWith("=>test");
+
+	proxy.error("test2");
+	expect(console.log).toHaveBeenLastCalledWith("=>test2");
+
+	proxy.info("test3");
+	expect(console.log).toHaveBeenLastCalledWith("=>test3");
+
+	proxy.debug("test4");
+	expect(console.log).toHaveBeenLastCalledWith("=>test4");
+
+	proxy.warn(() => "ftest");
+	expect(console.log).toHaveBeenLastCalledWith("=>ftest");
+
+	proxy.error(() => "ftest2");
+	expect(console.log).toHaveBeenLastCalledWith("=>ftest2");
+
+	proxy.info(() => "ftest3");
+	expect(console.log).toHaveBeenLastCalledWith("=>ftest3");
+
+	proxy.debug(() => "ftest4");
+	expect(console.log).toHaveBeenLastCalledWith("=>ftest4");
+
+	expect((proxy as any).prefix).toEqual("=>");
```

Comment:
I ended up refactoring out all of these and properly using the public interfaces, so resolving this now.

(or just removing some of these, such as this in case, where the behavior was _already_ tested via the public interface)